index.js

```diff
@@ -94,14 +101,22 @@ class ServiceClientFactory {
     });
   }
 
-  createServiceClient(serviceName, url, audience, tokenRefreshRate) {
+  createServiceClient(
+    serviceName,
+    url,
+    audience,
+    tokenRefreshRate,
+    getCorrelationId,
```

Comment:
Spoke virtually irl: This should be refactored into an options argument (along with `tokenRefreshRate` and maybe `audience`. As only fulfillment is using the BFFClient right now, the downstream shouldn't be too bad.